communications/rfc.rst

```diff
@@ -16,13 +16,32 @@ On this page:
 - :ref:`rfc-jira`.
 - :ref:`rfc-cases`.
 - :ref:`rfc-discussion`.
-- :ref:`rfc-exceptions`.
 - :ref:`rfc-formalities`.
 
 .. seealso::
 
    :doc:`rfd` has a similar name, but has to do with convening discussions across DM teams.
 
+.. _rfc-cases:
+
+Standard procedures that require an RFC
+=======================================
+
+The following procedures **always** require an RFC:
+
+- :doc:`/stack/adding-a-new-package`.
+- :doc:`/stack/packaging-third-party-eups-dependencies`, or with Conda.
+- :doc:`Updating change-controlled documentation </project-docs/change-controlled-docs>`.
+  An RFC can be used to directly obtain approval for modifying documents which are change-controlled at the DM level (those with “LDM-” handles).
+  An RFC should also be used to to develop subsystem-level consensus before submitting an LCR to modify documents which are change controlled at the project level (for example, those with “LSE-” or “LPM-” handles).
+- :doc:`/stack/deprecating-interfaces`, including changes to programming interfaces, configuration fields, and removal of packages.
+- Making changes to the data products which are or will be provided by the Data Management System.
+  This includes both changes to construction-era data products (such as intermediate stages in the evolution of an on-disk format) and to operations-era data products (which will normally be covered by an update to the baselined documentation describing the product).
+- Changes to certain procedures or policies described in this Developer Guide, including changes to coding standards.
```

Comment:
In _theory_ an RFC is not strictly necessary if I approve of the change, but in practice I agree that it is a good idea to go through one.  There are certain trivial corrections that should not need it.

Reply:
Yes, we have given K-T leeway for these in the past without requiring full DMCCB meeting.

Reply:
The [boilerplate on the coding standards](https://github.com/lsst-dm/dm_dev_guide/blame/master/cpp/style.rst#L10) says that changes should be requested by RFC, and I'd like to remain consistent with that, so I propose to leave this text as is for now.

Note that the next line of the text under review allows for changes to coding standards to be approved without be escalated to the DMCCB:

> In all except the last case above, the RFC must be approved by the DM Change Control Board ...